mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -23,11 +24,8 @@
 
 from pkg_resources import resource_filename
 
-# NEW TESTING DATA NEEDS TO BE GENERATED AFTER
-# REDUCING TESTING DATASET
-
-# Are any additional tests needed?
-# Top level?
+# NEW TESTING DATA NEEDS TO BE GENERATED FOR circstats
```

Comment:
I don't understand the comment.

If there's a TODO, raise an issue and reference the issue number if necessary. Otherwise remove comments. They will just be confusing later.